test/declaration_test.dart

```diff
@@ -774,6 +774,17 @@ div {
   expect(stylesheet != null, true);
   expect(errors.isEmpty, true, reason: errors.toString());
   expect(prettyPrint(stylesheet), generated3);
+
+  final input4 = '''
+div {
+  filter: FlipH;
+}''';
```

Comment:
How is this different then the tests for

Filter: Gray;

(see test in this file).  Is the issue one filter because filter: FlipH; should be parsed.  If its one filter then the probably is probably in a above parsing for RBRACE.